install_deps_release.sh

```diff
@@ -45,7 +45,8 @@ install_boost() {
     cd boost
     mkdir build
     cd build
-    cmake -DBOOST_INCLUDE_LIBRARIES="program_options;thread;locale;asio;lockfree;bimap" ..
+    #cmake -DBOOST_INCLUDE_LIBRARIES="program_options;thread;locale;asio;lockfree;bimap" ..
```

Comment:
remove comment

Reply:
I intentionally left this comment to recall how to install only necessary components